vite.config.preview.ts

```diff
@@ -46,6 +46,10 @@ export const config: (context: PreviewConfigContext) => UserConfigFn = ({
                 baseGuiPath: base,
                 apiUrl: api,
                 version,
+                product: 'Kuma',
+                mode: 'global',
+                environment: 'universal',
+                apiReadOnly: false,
```

Comment:
**Question**: Out of curiosity, can actually rely on this data being present in the index.html file already? I.e. is the backend work for this complete?

Reply:
I ran `kuma` and took these values from the HTML file it serves. The PR for this was done quite a while ago I believe, let me know if you want me to find the link for it.